IdleServer.js

```diff
@@ -52,11 +46,11 @@ const SERVER_CONFIG = {
 const PLAYFAB_ENVIRONMENT_CONFIG = {
 	configs: {
 		"18A45": {
-			apiKey: "C5NQN8YPPCOF43HSJEWOJNZ9Z7FI9IMSIZDGU7H1CZEMJTNZT6", // playfab auto
+			apiKey: "ORX7Y8EKSK5DD1MOKUE86X5D64RQ774359ICEJJMN7U176GOBF", // playfab auto
 			logglyTag: 'idle-spongebob-auto-ops'
 		},
 		"3D16B": {
-			apiKey: "1N6S3FUPY9UZPCPRHA5J1UR7TSM3UCA3AERMCIOZ6TFGD3693B",   // playfab kongregate
+			apiKey: "9QPQ1B5Q64G8Q3DAJE9WUFTC957UP3MZADYICBD97SWBXG853J",   // playfab kongregate
```

Comment:
I think we do not need this either, at least until we have to hit outside the server sdk or cloud script helper objects